.github/workflows/tag.yaml

```diff
@@ -105,10 +105,27 @@ jobs:
         fi
         make build-cli
         make helm-version
+
+   # GitIngest setup and run
+    - name: Set up Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: '3.10'
+
+    - name: Install GitIngest
+      run: pip install gitingest==0.4.2
```

Comment:
I just tried this locally and this release does not exist

```
➜  go git:(eitanya/adk) ✗ pip install gitingest==0.4.2
Defaulting to user installation because normal site-packages is not writeable
ERROR: Could not find a version that satisfies the requirement gitingest==0.4.2 (from versions: 0.1.1, 0.1.2, 0.1.3, 0.1.4, 0.1.5)

ERROR: No matching distribution found for gitingest==0.4.2
```

Reply:
oh, yeah, it is supported to be 0.1.2 , but will stick with the latest 0.1.5